benchmarks/power_distribution/power_distributor.py

```diff
@@ -105,12 +107,12 @@ async def run_test(  # pylint: disable=too-many-locals
     """
     start = timeit.default_timer()
 
-    power_request_channel = Broadcast[Request]("power-request")
+    active_power_request_channel = Broadcast[Request]("active_power-request")
     battery_status_channel = Broadcast[BatteryStatus]("battery-status")
-    channel_registry = ChannelRegistry(name="power_distributor")
+    channel_registry = ChannelRegistry(name="active_power_distributor")
```

Comment:
Also this probably needs to stay as `power_distributor`, as this is talking about an entity that has that name, but not sure what kind of power the power distributor distributes :D Also maybe @shsms or @mathias-baumann-frequenz know better.

Reply:
We are currently setting only the active power, so these are active power results.  We might start supporting setting reactive power,  and the new api specs already support it.

I'm actually doubtful of this whole exercise, because I imagine the maintenance of power factor to be a lower level function.  I'd rather just leave everything as it is,  and if we decide to introduce support for reactive power, call those methods `reactive`.